src/Symfony/Component/Filesystem/Tests/FilesystemTest.php

```diff
@@ -162,15 +164,23 @@ public function testCopyCreatesTargetDirectoryIfItDoesNotExist()
         $this->assertStringEqualsFile($targetFilePath, 'SOURCE FILE');
     }
 
-    /**
-     * @group network
-     */
     public function testCopyForOriginUrlsAndExistingLocalFileDefaultsToCopy()
     {
-        if (!\in_array('https', stream_get_wrappers())) {
-            $this->markTestSkipped('"https" stream wrapper is not enabled.');
+        if (!\in_array('http', stream_get_wrappers())) {
+            $this->markTestSkipped('"http" stream wrapper is not enabled.');
         }
-        $sourceFilePath = 'https://symfony.com/images/common/logo/logo_symfony_header.png';
+
+        $finder = new PhpExecutableFinder();
+        $process = new Process(array_merge([$finder->find(false)], $finder->findArguments(), ['-dopcache.enable=0', '-dvariables_order=EGPCS', '-S', '127.0.0.1:8057']));
+        $process->setWorkingDirectory(__DIR__.'/Fixtures/web');
+
+        $process->start();
```

Comment:
shouldn't we stop the process in a finally  block to ensure it is properly closed ?

Reply:
indeed, done